src/pages/NotePage/Notes.jsx

```diff
@@ -105,7 +119,7 @@ return (
             </section>
 
             <section className="note-card-container ">
-              {filteredNotes.map(({title,body,CreatedAt,backGround,_id,tag})=>{
+              {filteredNotes.map(({title,body,CreatedAt,backGround,_id,tag,actualTime})=>{
```

Comment:
`filteredNotes.map(({
       title,
       body,
       CreatedAt,
       backGround,
       _id,
       tag,
       actualTime
})=
>{`
this way of organizing is more readable usually